### DISCUSSION TITLE
Does not mention Small Visual Basic.

### DISCUSSION THREAD

**Discussion by wc-yaa** [VBAndCs/sVB-Small-Visual-Basic#74] [General]
I discovered this: https://litdev-uk.itch.io/

The webpage is quite nice.

However, the webpage only mentions Small Basic. In fact, the extension can also be used with Small Visual Basic. If you can make the author of the extension to mention Small Visual Basic on the webpage, the webpage will become an excellent promotion for Small Visual Basic. The reason is that a lot of teenagers and children like to visit Itch! Sure.

**Comment by VBAndCs** [comment]
The author is Steve Todman, and I can only contact him in the Small basic group on facebook. He already created a litDev version for sVB and I included it in its installer to be ready to use directly.
I will rely your suggestion to him.
Thanks.


________________________________
From: wc-yaa ***@***.***>
Sent: Sunday, July 21, 2024 10:26 PM
To: VBAndCs/sVB-Small-Visual-Basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VBAndCs/sVB-Small-Visual-Basic] Does not mention Small Visual Basic. (Discussion #74)


I discovered this: https://litdev-uk.itch.io/

The webpage is quite nice.

However, the webpage only mentions Small Basic. In fact, the extension can also be used with Small Visual Basic. If you can make the author of the extension to mention Small Visual Basic on the webpage, the webpage will become an excellent promotion for Small Visual Basic. The reason is that a lot of teenagers and children like to visit Itch! Sure.

**Reply by wc-yaa** [reply]
Oh! Doesn't want to help? Why?? 
Will you have time to plan to make an Itch page which is similar to LitDev's page?

**Comment by zs-3** [comment]
Hii @wc-yaa
You Can directly contact the author of litdev extension and that itch website Steve Todman on his small basic forum where intresting projects and works are going on and he is very active there.
Forum link : https://litdev.uk/mybb/
His Website : https://litdev.uk/

**Reply by wc-yaa** [reply]
Thanks. The information is useful.

**Reply by wc-yaa** [reply]
zs-3, 

Hey! I see this link:  https://zs-3.itch.io/

The webpage looks nice! Will you upload other Small Basic projects?

**Reply by zs-3** [reply]
Yes as I create new project I will upload

**Comment by VBAndCs** [comment]
No. I don't want to sign in in any more sites. I don't even want to activate the Github 2 factor authentication, and I will not update the source code here anymore. I will just update the installer on VS market place.
Also I don't understand Itch and it seems to allow sexual games.
sVB may help spread it by all means available. The missing ley here is creating YouTube tutorials about sVB in English and other languages. I did my share by creating them in Arabic on
***@***.***
But I can't do more.
Thanks.

________________________________

**Reply by zs-3** [reply]
Why will you not update source code

**Reply by zs-3** [reply]
I guess you can't login

**Reply by zs-3** [reply]
And I had a question I was trying to make SVB GW work in sb using my extension it works but I didn't updated extension I don't know SVB licence could I continue this project as SVB is very powerful and it will make sb also more powerful

**Comment by VBAndCs** [comment]
Yes I has issues with the 2 factor and after my SSD is dead I failed to use the command line to update the source. Also, I got no code contributions, so I don't find any goal of continuing publishing the source code.
sVB is based on SB and both are MIT. Feel free to use sVB as you want. You can create versions of your extensions for sVB by targeting SmallVisualBasicLibrary.dll instead of SmallBasicLibrary.dll
________________________________
